src/portofolio_sections/AboutContent.jsx

```diff
@@ -95,3 +102,7 @@ const P = styled.p`
   font-size: 20px;
   line-height: 2rem;
 `;
+
+const Frame = styled.div`
+  height: 100vh;
```

Comment:
making every division's height equal to the screen height is not a good decision.

It introduces unnecessary extra space.
![image](https://user-images.githubusercontent.com/31813384/95343114-c7ea2800-08d5-11eb-9256-779e4f243db2.png)